libconfig/src/config_content_node.c

```diff
@@ -186,6 +188,50 @@ const char * config_content_node_iget_as_relpath( config_content_node_type * nod
   }
 }
 
+const char * config_content_node_iget_as_executable( config_content_node_type * node , int index) {
+  config_schema_item_assure_type(node->schema , index ,
+        CONFIG_PATH + CONFIG_EXISTING_PATH + CONFIG_EXECUTABLE );
+  {
+    const char * config_value = config_content_node_iget(node , index);
+    const char * separator = strstr( config_value, "/" );
+
+    /* possible binary-name, consult PATH */
+    char* path_value = NULL;
+    const char* PATH = getenv( "PATH" );
+    if( !separator && PATH ) {
```

Comment:
There is some functionality quite close (I think) to this as`util_alloc_PATH_executable`